Flaky Test - `TestEncryptionDecryption`

# Problem

```bash
➜  status-go git:(chore/test-request-community-from-fleet) ✗ go test -v ./protocol/communities -run ^TestCommunityEncryptionDescriptionSuite$ -testify.m TestEncryptionDecryption -test.count=30
=== RUN   TestCommunityEncryptionDescriptionSuite
=== RUN   TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
--- PASS: TestCommunityEncryptionDescriptionSuite (0.00s)
    --- PASS: TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption (0.00s)
=== RUN   TestCommunityEncryptionDescriptionSuite
=== RUN   TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
--- PASS: TestCommunityEncryptionDescriptionSuite (0.00s)
    --- PASS: TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption (0.00s)
=== RUN   TestCommunityEncryptionDescriptionSuite
=== RUN   TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
    community_description_encryption_test.go:129: 
                Error Trace:    /Users/igorsirotin/Repositories/Status/status-desktop-3/vendor/status-go/protocol/communities/community_description_encryption_test.go:129
                Error:          "map[985f4af2-0c2a-45fa-a236-40b6fd8e5e73:encryptedDescription]" should have 2 item(s), but has 1
                Test:           TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
--- FAIL: TestCommunityEncryptionDescriptionSuite (0.00s)
    --- FAIL: TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption (0.00s)
=== RUN   TestCommunityEncryptionDescriptionSuite
=== RUN   TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
--- PASS: TestCommunityEncryptionDescriptionSuite (0.00s)
    --- PASS: TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption (0.00s)
=== RUN   TestCommunityEncryptionDescriptionSuite
=== RUN   TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
--- PASS: TestCommunityEncryptionDescriptionSuite (0.00s)
    --- PASS: TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption (0.00s)
=== RUN   TestCommunityEncryptionDescriptionSuite
=== RUN   TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
    community_description_encryption_test.go:129: 
                Error Trace:    /Users/igorsirotin/Repositories/Status/status-desktop-3/vendor/status-go/protocol/communities/community_description_encryption_test.go:129
                Error:          "map[bc567f39-1942-462c-96f0-d247b7beacb8:encryptedDescription]" should have 2 item(s), but has 1
                Test:           TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
--- FAIL: TestCommunityEncryptionDescriptionSuite (0.00s)
    --- FAIL: TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption (0.00s)
=== RUN   TestCommunityEncryptionDescriptionSuite
=== RUN   TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
--- PASS: TestCommunityEncryptionDescriptionSuite (0.00s)
    --- PASS: TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption (0.00s)
=== RUN   TestCommunityEncryptionDescriptionSuite
=== RUN   TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
--- PASS: TestCommunityEncryptionDescriptionSuite (0.00s)
    --- PASS: TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption (0.00s)
=== RUN   TestCommunityEncryptionDescriptionSuite
=== RUN   TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
--- PASS: TestCommunityEncryptionDescriptionSuite (0.00s)
    --- PASS: TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption (0.00s)
=== RUN   TestCommunityEncryptionDescriptionSuite
=== RUN   TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
--- PASS: TestCommunityEncryptionDescriptionSuite (0.00s)
    --- PASS: TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption (0.00s)
=== RUN   TestCommunityEncryptionDescriptionSuite
=== RUN   TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
--- PASS: TestCommunityEncryptionDescriptionSuite (0.00s)
    --- PASS: TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption (0.00s)
=== RUN   TestCommunityEncryptionDescriptionSuite
=== RUN   TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
    community_description_encryption_test.go:129: 
                Error Trace:    /Users/igorsirotin/Repositories/Status/status-desktop-3/vendor/status-go/protocol/communities/community_description_encryption_test.go:129
                Error:          "map[a3c5d59e-e13d-498f-ab7a-9ce40f28a7dc:encryptedDescription]" should have 2 item(s), but has 1
                Test:           TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
--- FAIL: TestCommunityEncryptionDescriptionSuite (0.00s)
    --- FAIL: TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption (0.00s)
=== RUN   TestCommunityEncryptionDescriptionSuite
=== RUN   TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
--- PASS: TestCommunityEncryptionDescriptionSuite (0.00s)
    --- PASS: TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption (0.00s)
=== RUN   TestCommunityEncryptionDescriptionSuite
=== RUN   TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
--- PASS: TestCommunityEncryptionDescriptionSuite (0.00s)
    --- PASS: TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption (0.00s)
=== RUN   TestCommunityEncryptionDescriptionSuite
=== RUN   TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
--- PASS: TestCommunityEncryptionDescriptionSuite (0.00s)
    --- PASS: TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption (0.00s)
=== RUN   TestCommunityEncryptionDescriptionSuite
=== RUN   TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
--- PASS: TestCommunityEncryptionDescriptionSuite (0.00s)
    --- PASS: TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption (0.00s)
=== RUN   TestCommunityEncryptionDescriptionSuite
=== RUN   TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
--- PASS: TestCommunityEncryptionDescriptionSuite (0.00s)
    --- PASS: TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption (0.00s)
=== RUN   TestCommunityEncryptionDescriptionSuite
=== RUN   TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
    community_description_encryption_test.go:129: 
                Error Trace:    /Users/igorsirotin/Repositories/Status/status-desktop-3/vendor/status-go/protocol/communities/community_description_encryption_test.go:129
                Error:          "map[140f5ed4-4e72-4cb0-a794-fcf9f466413f:encryptedDescription]" should have 2 item(s), but has 1
                Test:           TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
--- FAIL: TestCommunityEncryptionDescriptionSuite (0.00s)
    --- FAIL: TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption (0.00s)
=== RUN   TestCommunityEncryptionDescriptionSuite
=== RUN   TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
    community_description_encryption_test.go:129: 
                Error Trace:    /Users/igorsirotin/Repositories/Status/status-desktop-3/vendor/status-go/protocol/communities/community_description_encryption_test.go:129
                Error:          "map[b396908d-e5be-4b3b-8a6d-f3b936c5ef9a:encryptedDescription]" should have 2 item(s), but has 1
                Test:           TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
--- FAIL: TestCommunityEncryptionDescriptionSuite (0.00s)
    --- FAIL: TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption (0.00s)
=== RUN   TestCommunityEncryptionDescriptionSuite
=== RUN   TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
    community_description_encryption_test.go:129: 
                Error Trace:    /Users/igorsirotin/Repositories/Status/status-desktop-3/vendor/status-go/protocol/communities/community_description_encryption_test.go:129
                Error:          "map[8fa22ee7-669b-4302-9356-d7e089c9fe4c:encryptedDescription]" should have 2 item(s), but has 1
                Test:           TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
--- FAIL: TestCommunityEncryptionDescriptionSuite (0.00s)
    --- FAIL: TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption (0.00s)
=== RUN   TestCommunityEncryptionDescriptionSuite
=== RUN   TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
--- PASS: TestCommunityEncryptionDescriptionSuite (0.00s)
    --- PASS: TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption (0.00s)
=== RUN   TestCommunityEncryptionDescriptionSuite
=== RUN   TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
    community_description_encryption_test.go:129: 
                Error Trace:    /Users/igorsirotin/Repositories/Status/status-desktop-3/vendor/status-go/protocol/communities/community_description_encryption_test.go:129
                Error:          "map[39bd7942-51b3-42a9-a517-571b77fdcb73:encryptedDescription]" should have 2 item(s), but has 1
                Test:           TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
--- FAIL: TestCommunityEncryptionDescriptionSuite (0.00s)
    --- FAIL: TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption (0.00s)
=== RUN   TestCommunityEncryptionDescriptionSuite
=== RUN   TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
    community_description_encryption_test.go:129: 
                Error Trace:    /Users/igorsirotin/Repositories/Status/status-desktop-3/vendor/status-go/protocol/communities/community_description_encryption_test.go:129
                Error:          "map[40118acb-3fa0-45f0-a74c-03b2e8174efa:encryptedDescription]" should have 2 item(s), but has 1
                Test:           TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
--- FAIL: TestCommunityEncryptionDescriptionSuite (0.00s)
    --- FAIL: TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption (0.00s)
=== RUN   TestCommunityEncryptionDescriptionSuite
=== RUN   TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
--- PASS: TestCommunityEncryptionDescriptionSuite (0.00s)
    --- PASS: TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption (0.00s)
=== RUN   TestCommunityEncryptionDescriptionSuite
=== RUN   TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
--- PASS: TestCommunityEncryptionDescriptionSuite (0.00s)
    --- PASS: TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption (0.00s)
=== RUN   TestCommunityEncryptionDescriptionSuite
=== RUN   TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
    community_description_encryption_test.go:129: 
                Error Trace:    /Users/igorsirotin/Repositories/Status/status-desktop-3/vendor/status-go/protocol/communities/community_description_encryption_test.go:129
                Error:          "map[f10dcdf8-66cf-4a45-90ca-b4273d9cccdf:encryptedDescription]" should have 2 item(s), but has 1
                Test:           TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
--- FAIL: TestCommunityEncryptionDescriptionSuite (0.00s)
    --- FAIL: TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption (0.00s)
=== RUN   TestCommunityEncryptionDescriptionSuite
=== RUN   TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
--- PASS: TestCommunityEncryptionDescriptionSuite (0.00s)
    --- PASS: TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption (0.00s)
=== RUN   TestCommunityEncryptionDescriptionSuite
=== RUN   TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
--- PASS: TestCommunityEncryptionDescriptionSuite (0.00s)
    --- PASS: TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption (0.00s)
=== RUN   TestCommunityEncryptionDescriptionSuite
=== RUN   TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
--- PASS: TestCommunityEncryptionDescriptionSuite (0.00s)
    --- PASS: TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption (0.00s)
=== RUN   TestCommunityEncryptionDescriptionSuite
=== RUN   TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
    community_description_encryption_test.go:129: 
                Error Trace:    /Users/igorsirotin/Repositories/Status/status-desktop-3/vendor/status-go/protocol/communities/community_description_encryption_test.go:129
                Error:          "map[48edc561-dd82-40d9-a124-623fd019e727:encryptedDescription]" should have 2 item(s), but has 1
                Test:           TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption
--- FAIL: TestCommunityEncryptionDescriptionSuite (0.00s)
    --- FAIL: TestCommunityEncryptionDescriptionSuite/TestEncryptionDecryption (0.00s)
FAIL
FAIL    github.com/status-im/status-go/protocol/communities     0.552s
FAIL
```

--- a/protocol/communities/community_description_encryption_test.go
+++ b/protocol/communities/community_description_encryption_test.go
@@ -110,8 +110,8 @@ func (s *CommunityEncryptionDescriptionSuite) description() *protobuf.CommunityD
 				ChatIds:       []string{},
 			},
 			"channel-level-permission": &protobuf.CommunityTokenPermission{
-				Id:            "community-level-permission",
-				Type:          protobuf.CommunityTokenPermission_BECOME_MEMBER,
+				Id:            "channel-level-permission",
+				Type:          protobuf.CommunityTokenPermission_CAN_VIEW_CHANNEL,
 				TokenCriteria: []*protobuf.TokenCriteria{},
 				ChatIds:       []string{types.EncodeHex(crypto.CompressPubkey(&s.identity.PublicKey)) + "channelB"},
 			},

